body/stretch_body/device.py

```diff
@@ -93,10 +93,24 @@ def pretty_print(self):
         print('----- {0} ------ '.format(self.name))
         hello_utils.pretty_print_dict("params", self.params)
 
-    def write_device_params(self,device_name, params,fleet_dir=None):
-        rp=hello_utils.read_fleet_yaml(self.user_params['factory_params'],fleet_dir=fleet_dir)
-        rp[device_name]=params
-        hello_utils.write_fleet_yaml(self.user_params['factory_params'],rp,fleet_dir=fleet_dir)
```

Comment:
Although it's unlikely that anyone has used this method besides us, we should keep it and raise a deprecation warning rather than delete it outright.

```
def write_device_params(self):
    raise DeprecationWarning('This method has been deprecated since v0.3.0')
```

Deprecated methods can stick around in this way for a few months to give everyone a chance to see it, and then be deleted from the code base.

Reply:
Done